### ISSUE TITLE
Unable to start apache httpd after adding auth_openidc_module module

### ISSUE THREAD

**Issue by akkond** [zmartzone/mod_auth_openidc#387]
Hello,

Installed mod_auth_openidc 2.3.7-1 and all its dependencies using rpm on RHEL7 64 bit.
Configured Apache httpd.conf with below details 
LoadFile /usr/lib64/libcjose.so
LoadFile /usr/lib64/libjansson.so
LoadModule auth_openidc_module  /usr/lib64/httpd/modules/mod_auth_openidc.so

When i try to start apache getting below error
./apachectl -k start
httpd: Syntax error on line 920 of /conf/httpd.conf: Cannot load /usr/lib64/httpd/modules/mod_auth_openidc.so into server: /usr/lib64/httpd/modules/mod_auth_openidc.so: undefined symbol: apr_memcache_add_server

Are there any other dependencies i am missing.

###### Environment

- mod_auth_openidc version 2.3.7-1.el7
- Apache version 2.2.8
- platform/distro RHEL7 64-bit

**Comment by zandbelt** [comment]
`apr` and `apr-util` should have been installed as dependencies; can you past the output of:
```
yum list installed | grep apr
```
which should be somewhat similar to the output on CentOS 7:
```
apr.x86_64                                1.5.2-1.el7.codeit           @CodeIT  
apr-util.x86_64                           1.5.2-6.el7                  @base   
```

**Comment by zandbelt** [comment]
Also, you seen to have Apache 2.2.8 but the default version should be:
```
RHEL 7: based on upstream v2.4.6
```
see: https://access.redhat.com/solutions/445713

You also shouldn't need the `LoadFile` directives, not sure where those came from.

It seems you're on a heavily customized version of RHEL 7?

**Comment by zandbelt** [comment]
Any progress with this?

**Comment by akkond** [comment]
Thanks Zandbelt for your quick response. 
I am trying to integrate mod_auth_openidc into IBM Http server 8.5.5 which is built on Apache 2.2.8.

I tried to install mod_auth_openidc RHEL 6.9 but failed due to its dependency on hiredis module which wasnt available for RHEL 6.5 so chose RHEL 7.

Both above packages are already installed but still get the /usr/lib64/httpd/modules/mod_auth_openidc.so: undefined symbol: apr_memcache_add_server
 yum list installed | grep apr
apr.x86_64                 1.4.8-3.el7_4.1     @rhui-REGION-rhel-server-releases
apr-util.x86_64            1.5.2-6.el7         @rhui-REGION-rhel-server-releases

LoadFile directives i added manually after looking at similar startup issues.

**Comment by zandbelt** [comment]
The combination of IBM http server and mod_auth_openidc is not supported by default nor are the packages built for any other web server than the one that comes with the stock distribution. If you need this you have to engage in a commercial support/development process via sales@zmartzone.eu.
